### PR TITLE
Moved astropy.vo.samp to astropy.samp

### PR DESCRIPTION
**Pull request by astrofrog**
@taldcroft and @mhvk agreed to this plan in https://github.com/astropy/astropy/pull/6201

@eteq @kelle @pllim - any objections to this?

@pllim - I actually ended up just removing ``docs/vo`` and adding the note you had added about astroquery to the what's new instead. Does that seem ok?

### CODE REVIEW

**Comment by pllim** [comment]
Wait a minute... Are you removing `astropy.vo` without deprecation? I don't plan to remove Cone Search until v3; I just deprecated it for v2...

File: astropy/astropy.cfg [hunk 1]
```diff
@@ -62,7 +62,7 @@
 
 [vo.samp]
```
Comment (pllim) [inline astropy/astropy.cfg]: Don't you need a new `[samp]` section too?

**Comment by astrofrog** [comment]
@pllim - no I've only removed ``docs/vo`` not ``astropy.vo`` - that is, the imports will continue to work but the docs will go away. But I'm happy to add back the docs if you'd prefer to remove that section after 2.0.

**Comment by pllim** [comment]
@astrofrog , moving doc only is probably okay, but the diff also shows you moved the codes and deleted some cone search stuff.

**Comment by astrofrog** [comment]
@pllim - I just moved the files up one directory, and ``astropy/vo/samp/__init__.py`` now imports from ``astropy.samp`` so it's backward compatible. I didn't touch any of the cone search stuff as far as I know?

**Comment by pllim** [comment]
@astrofrog , okay, I was just freaking out of all the cone search doc deletion. :sweat_smile: 

It just seems confusing to a user that they can still use `astropy.vo` but the documentation is gone. Maybe retain a `vo/index.rst` with the text you put in What's New? Not all users will dig into What's New.

Also need to update the "stability" listing.

**Comment by astrofrog** [comment]
@pllim - I've undeleted the vo docs!

**Comment by pllim** [comment]
Thanks! Still need a change log and edits to `docs/stability.rst`.

**Comment by astrofrog** [comment]
@pllim - done!

**Comment by astrofrog** [comment]
Before we merge this, just a reminder that I'd like to make sure @eteq is happy with this

File: CHANGES.rst [hunk 2]
```diff
@@ -275,6 +275,8 @@ API Changes
 
   - The `astropy.vo.samp` package no longer supports SSL. [#6201]
 
+  - The `astropy.vo.samp` package has been moved to `astropy.samp`. [#6213]
```
Comment (pllim) [inline CHANGES.rst]: Nitpick: This entry and the one for #6201 above are a bit confusing. Maybe merge them into a single entry and reword in a more coherent manner?

File: docs/stability.rst [hunk 3]
```diff
@@ -298,7 +298,7 @@ The current planned and existing sub-packages are:
         </tr>
         <tr>
             <td>
-                astropy.vo.samp
+                astropy.samp
```
Comment (pllim) [inline docs/stability.rst]: Maybe it is clearer to state in the description that it was renamed from ``astropy.vo.samp`` in v2.0? Other entries have info like that (when added, etc).

Comment (astrofrog) [inline docs/stability.rst]: Done

File: CHANGES.rst (hunk 2 above)
Comment (astrofrog) [inline CHANGES.rst]: Done

**Comment by bsipocz** [comment]
Added @eteq as a reviewer, as he needs to be happy with this move :)

File: docs/vo/index.rst [hunk 4]
```diff
@@ -6,19 +6,20 @@ Virtual Observatory Access (``astropy.vo``)
...
 .. module:: astropy.vo
 
-Introduction
-============
+.. warning::
+
+    The ``astropy.vo`` module has been deprecated and will be removed
+    in a future version. The SAMP functionality has been moved
+    to ``astropy.samp``, and the conesearch functionality has been
+    moved to `Astroquery <http://astroquery.readthedocs.io>`_.
```
Comment (eteq) [inline docs/vo/index.rst]: Shouldn't this also mention that this is the "classic" API?

Comment (pllim) [inline docs/vo/index.rst]: @eteq , are you talking about Cone Search or SAMP's "classic" API?

File: CHANGES.rst [hunk 5]
```diff
@@ -273,7 +273,8 @@ API Changes
     Astroquery 0.3.5 and will be removed from Astropy in a future version.
     [#5558, #5904]
 
-  - The `astropy.vo.samp` package no longer supports SSL. [#6201]
+  - The `astropy.vo.samp` package has been moved to `astropy.samp`, and no
```
Comment (bsipocz) [inline CHANGES.rst]: use double backticks to make sphinx happy

File: docs/vo/index.rst (hunk 4 above)
Comment (eteq) [inline docs/vo/index.rst]: Oops!  Sorry, misunderstanding: I thought the what's new page was saying that there "classic" API is in astropy, but I get now that that's referencing the astroquery version.  So no change needed.

**Comment by eteq** [comment]
Looks like the tests passed except for that one backtick issue that I fixed directly in this branch.  Merging!

**Pull request merged**